include/exadg/grid/grid.cpp

```diff
@@ -233,15 +247,22 @@ Grid<dim>::do_create_triangulation(
 
     unsigned int const group_size = 1;
 
+    typename dealii::TriangulationDescription::Settings triangulation_description_setting;
+
+    if(data.create_coarse_triangulations)
+      triangulation_description_setting = dealii::TriangulationDescription::default_setting;
+    else
+      triangulation_description_setting =
+        dealii::TriangulationDescription::construct_multigrid_hierarchy;
+
     // TODO SIMPLEX: this will not work in case of simplex meshes
     auto const description = dealii::TriangulationDescription::Utilities::
-      create_description_from_triangulation_in_groups<dim, dim>(
-        serial_grid_generator,
-        serial_grid_partitioner,
-        triangulation->get_communicator(),
-        group_size,
-        dealii::Triangulation<dim>::none,
-        dealii::TriangulationDescription::construct_multigrid_hierarchy);
+      create_description_from_triangulation_in_groups<dim, dim>(serial_grid_generator,
+                                                                serial_grid_partitioner,
+                                                                triangulation->get_communicator(),
+                                                                group_size,
+                                                                mesh_smoothing,
+                                                                triangulation_description_setting);
```

Comment:
this will change the code in my opinion. Is this intended?

Reply:
Yes, the reason behind this is the same as setting the distributed triangulation setting to the default setting. If construct multigrid hierarchy is activated, limit level differences at vertices is set automatically: https://github.com/dealii/dealii/blob/69313620a085f92bc4f8e619424eeeba47927840/source/grid/tria_description.cc#L822 

In both cases the code changes.